app/views/admin/admin/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<%= render "homepage/header" %>
+<%= render "admintab" %>
+<div class="container">
+  <h2>WorkSpace</h2>            
+  <table class="table table-bordered">
+   
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Email</th>
+        <th>Gender</th>
+        <th>status</th>
+        <th>Manegerial Rights</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @user.each do |user|%>
+        <% if !user.admin? %>
```

Comment:
you can use 
<% next if user.admin? %>

instead of using the block.

Reply:
Fixed.